api/equality/equality.go

```diff
@@ -27,3 +30,50 @@ func TaskStatusesEqualStable(a, b *api.TaskStatus) bool {
 	copyA.Timestamp, copyB.Timestamp = nil, nil
 	return reflect.DeepEqual(&copyA, &copyB)
 }
+
+// RootCAEqualStable compares RootCAs, excluding join tokens, which are randomly generated
+func RootCAEqualStable(a, b *api.RootCA) bool {
+	switch {
+	case a == nil && b == nil:
+		return true
+	case a != nil && b != nil:
+	default:
+		return false
+	}
```

Comment:
This is super nitpicky, but as a former C programmer I find this confusing. How about:

```go
if a == nil && b == nil {
        return true
}
if a == nil || b == nil {
        return false
}
```